source/includes/inkling-reference/_concepts.html.md

```diff
@@ -59,20 +47,48 @@ concept
 end
 
 inputSource ::=
-    input '(' schemaRef? ')'
-  | <name>       # name of a concept or stream
+    input '(' schemaRef? ')' | <name>     # name of a concept or stream
 
 outputTarget ::=
-    output
- |  <name>       # name of a concept or stream
+    output |  <name>                      # name of a concept or stream
 ```
 
-Select the Syntax tab to show syntax for the concept statement and its input
-sources and output targets.
+### Usage
```

Comment:
I'll go ahead and make this change while I'm doing a couple small formatting changes.